### DISCUSSION TITLE
how can i Center an (existing) Floating window

### DISCUSSION THREAD

**Discussion by Kallinteris-Andreas** [i3/i3#5418] [Q&A] [answered]
Scenario:
I have a single floating window on my current workspace

What I want to do:
place that floating window at the center of the screen with a keyboard shortcut

Note:
I am not looking for a way to center the floating window during the creation of the floating window

Thanks!

Additionally:
is it possible to center the window on the vertical axis (without moving it horizontally)

**Comment by Airblader** [comment, marked as answer]
https://i3wm.org/docs/userguide.html#move_direction has the answer.
